onmt/train/Trainer.lua

```diff
@@ -77,6 +77,10 @@ local options = {
     [[When using translation-based validation metrics (e.g. BLEU, TER, etc.), also save the
     translation every this many epochs to the file `<save_model>_epochN_validation_translation.txt`.
     If = 0, will not save validation translation.]]
+  },
+  {
+    '-update_vocab', true,
```

Comment:
To not surprise existing users, this option should be disabled by default. Also update the documentation accordingly.